js_modules/dagster-ui/packages/ui-core/src/sensors/SensorDetails.tsx

```diff
@@ -153,6 +154,14 @@ export const SensorDetails = ({
               <td>{sensor.description}</td>
             </tr>
           ) : null}
+          {sensor.owners && sensor.owners.length > 0 && (
```

Comment:
Looks like `owners` is a non-nullable array, so you should just be able to check `sensor.owners.length > 0`.